airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -1,49 +1,62 @@
 connector_image: airbyte/source-stripe:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
-    - spec_path: "source_stripe/spec.yaml"
+    tests:
+      - spec_path: "source_stripe/spec.yaml"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "secrets/connected_account_config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "0.1.37"
-    - config_path: "secrets/connected_account_config.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "0.1.37"
+    tests:
+      - config_path: "secrets/config.json"
+        backward_compatibility_tests_config:
+          disable_for_version: "1.0.1"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/full_refresh_configured_catalog.json"
-      empty_streams: ["external_account_bank_accounts"]
-    # TEST 1 - Reading catalog without invoice_line_items
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/non_invoice_line_items_catalog.json"
-      timeout_seconds: 3600
-    # TEST 2 - Reading data from account that has no records for stream Disputes
-    - config_path: "secrets/connected_account_config.json"
-      configured_catalog_path: "integration_tests/connected_account_configured_catalog.json"
-      timeout_seconds: 3600
+    tests:
+      - config_path: "secrets/config.json"
+        empty_streams:
+          - name: "bank_accounts"
+            bypass_reason: "This stream can't be seeded in our sandbox account"
+          - name: "external_account_bank_accounts"
+            bypass_reason: "This stream can't be seeded in our sandbox account"
+          - name: "external_account_cards"
+            bypass_reason: "This stream can't be seeded in our sandbox account"
+          - name: "checkout_sessions"
+            bypass_reason: "This stream can't be seeded in our sandbox account"
+          - name: "checkout_sessions_line_items"
+            bypass_reason: "This stream can't be seeded in our sandbox account"
+          - name: "transfers"
+            bypass_reason: "This stream can't be seeded in our sandbox account"
+          - name: "charges"
+            bypass_reason: "This stream produces unique data every time it runs. Fields: receipt_url"
```

Comment:
This is the most important stream in the connector, and I feel uncomfortable certifying this connector to GA bypassing a test on this stream.  It's probably time to adjust SAT so that `basic-read` can ignore a singe column that is changing, like it is here.  This is a great example of a reasonable source changing data on every read (a protected PDF download link for a receipt is only valid for a short amount of time)



@roman-yermilov-gl - do you feel comfortable working on https://github.com/airbytehq/airbyte/issues/21737 to unblock this? 

cc @YowanR as I'm rejecting this, and cc @alafanechere who may have some advice on how to work on https://github.com/airbytehq/airbyte/issues/21737

Reply:
**do you feel comfortable working on https://github.com/airbytehq/airbyte/issues/21737 to unblock this?** - yes I do

Reply:
I replied in #21737 . I would also be interested in trying out a different approach: what would happen if you remove this field from the stream schema at the configured catalog level and remove it from the expected records?  It'll probably be considered as an additional property that'll be ignored if you set `extra_fields: true` in the expected records config.

Reply:
It looks like a kind of workaround for current case, but we need a solution for many other connectors that fail because of `expected_records`.